var/spack/repos/builtin/packages/intel-parallel-studio/package.py

```diff
@@ -148,6 +148,13 @@ class IntelParallelStudio(IntelPackage):
     conflicts('+vtune',     when='@composer.0:composer.9999')
 
     @property
+    def intel_prefix(self):
+        if self.version[0] == "composer" and self.version[1] == 2015:
```

Comment:
Trust me, my way of debugging spack is `raise RuntimeError(....)`.  The `type(self.version[1])` was definitely `int` xD